features/flows/rest/api/src/main/java/org/opennms/netmgt/flows/rest/FlowRestService.java

```diff
@@ -138,4 +138,15 @@ FlowSeriesResponse getTopNConversationsSeries(
             @Context final UriInfo uriInfo
     );
 
+    /**
+     * Retrieves the deep dive tool url from configured properties.
+     *
+     *
+     * @param uriInfo JAX-RS context
+     * @return url to deep dive tool
+     */
```

Comment:
The method name and `@Context` annotation is pretty straight forward. I'd opt for not repeating that with a comment.